### PR TITLE
Seo first part

### PR DESCRIPTION
**Pull request by marianorod**
About issue https://github.com/bitpay/bitcore/issues/1142

### CODE REVIEW

File: themes/bitcore/layout/index.ejs [hunk 1]
```diff
@@ -9,6 +9,7 @@
   <div class="outer">
     <div class="inner">
       <h2 id="banner-title">A pure and powerful JavaScript Bitcoin library.</h2>
+      <h2 id="banner-subtitle">Take advatage of the blockchain with a clean and simple API</h2>
```
Comment (martindale) [inline themes/bitcore/layout/index.ejs]: s/advatage/advantage

s/clean and simple/simple and clean/

**Comment by martindale** [comment]
Hey guys, I'll take a quick pass on this tonight or tomorrow morning.

File: themes/bitcore/layout/index.ejs (hunk 1 above)
Comment (braydonf) [inline themes/bitcore/layout/index.ejs]: We should mention "Bitcoin" and "JavaScript" and "library" here, what's the reasoning for removing those?

**Comment by braydonf** [comment]
Outdated, closing. We can reference later if it's needed.

**Pull request closed without merging**